distribution/lib/Standard/Base/0.2.27-SNAPSHOT/src/Data/Text/Extensions.enso

```diff
@@ -88,13 +92,231 @@ Text.characters =
     this.each bldr.append
     bldr.to_vector
 
-## ALIAS Split
+## ALIAS Match Text
+
+   Matches the text in `this` against the provided regex `pattern`, returning
+   the match(es) if present or `Nothing` if there are no matches.
+
+   Arguments:
+   - pattern: The pattern to match `this` against. We recommend using _raw text_
+     to write your patterns.
+   - mode: This argument specifies how many matches the engine will try and
+     find. When mode is set to either `Mode.First` or `Mode.Full`, this method
+     will return either a single `Match` or `Nothing`. If set to an `Integer` or
+     `Mode.All`, this method will return either a `Vector Match` or `Nothing`.
+   - match_ascii: Enables or disables pure-ASCII matching for the regex. If you
+     know your data only contains ASCII then you can enable this for a
+     performance boost on some regex engines.
+   - case_insensitive: Enables or disables case-insensitive matching. Case
+     insensitive matching behaves as if it normalises the case of all input
+     text before matching on it.
+   - dotall: Enables or disables the dotall option. Dotall specifies that the
+     `.` special character should match everything _including_ newline
+     characters. Without this flag, it will match all characters _except_
+     newlines.
+   - multiline: Enables or disables the multiline option. Multiline specifies
+     that the `^` and `$` pattern characters match the start and end of lines,
+     as well as the start and end of the input respectively.
+   - verbose: Enables or disables the verbose mode for the regular expression.
+     In verbose mode, the following changes apply:
+     - Whitespace within the pattern is ignored, except when within a
+       character class or when preceeded by an unescaped backslash, or within
+       grouping constructs (e.g. `(?...)`).
+     - When a line contains a `#`, that is not in a character class and is not
+       preceeded by an unescaped backslash, all characters from the leftmost
+       such `#` to the end of the line are ignored. That is to say, they act
+       as _comments_ in the regex.
+   - extra_opts: Specifies additional options in a vector. Where these overlap
```

Comment:
I think some specification is required. Also, what are these options? Why can't we make them arguments?